src/tasks/01_getMaxNumber.js

```diff
@@ -9,3 +9,16 @@
  */
 
 // <-- Тут реализация функции getMaxNumber
+function getMaxNumber(firstNumber, secondNumber) {
+    if (isNumber(firstNumber) && isNumber(secondNumber)) {
+        if (firstNumber < secondNumber) return secondNumber;
+        return firstNumber;
```

Comment:
Такие конструкции можно упрощать с помощью тернарного оператора
```suggestion
        return firstNumber < secondNumber ? secondNumber : firstNumber;
```